package.json

```diff
@@ -62,6 +62,7 @@
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prettier": "^3.1.4",
     "express": "^4.17.1",
+    "fastify": "^2.13.0",
```

Comment:
Latest version is `3.3.0` https://github.com/fastify/fastify/blob/master/package.json#L3, we don't need support old versions